CONTRIBUTING.md

```diff
@@ -74,91 +124,125 @@ optimize:
+Editor     | Plugin
+-----------|-------------------------------------------------------------------------------
+Sublime    | [SublimeLinter-eslint](https://github.com/roadhump/SublimeLinter-eslint#installation)
+Atom       | [linter-eslint](https://github.com/AtomLinter/linter-eslint#installation)
+IntelliJ   | Settings » Languages & Frameworks » JavaScript » Code Quality Tools » ESLint
+`vi`       | [scrooloose/syntastic](https://github.com/scrooloose/syntastic)
 
-Another tool we use for enforcing consistent coding style is Editorconfig, which can be set up by installing a plugin in your editor that dynamically updates its configuration. Take a look at the [Editorconfig](http://editorconfig.org/#download) site to find a plugin for your editor, and browse our [`.editorconfig`](https://github.com/elastic/kibana/blob/master/.editorconfig) file to see what config rules we set up.
+Another tool we use for enforcing consistent coding style is EditorConfig, which can be set up by installing a plugin in your editor that dynamically updates its configuration. Take a look at the [EditorConfig](http://editorconfig.org/#download) site to find a plugin for your editor, and browse our [`.editorconfig`](https://github.com/elastic/kibana/blob/master/.editorconfig) file to see what config rules we set up.
 
-### Testing and building
+### Testing and Building
 
-To ensure that your changes will not break other functionality, please run the test suite and build process before submitting your pull request.
+To ensure that your changes will not break other functionality, please run the test suite and build process before submitting your Pull Request.
 
 Before running the tests you will need to install the projects dependencies as described above.
 
-Once that is complete just run:
+Once that's done, just run:
 
-```
-sh
+```bash
 npm run test && npm run build -- --skip-os-packages
 ```
 
-#### Debugging unit tests
+### Debugging Unit Tests
 
 The standard `npm run test` task runs several sub tasks and can take several minutes to complete, making debugging failures pretty painful. In order to ease the pain specialized tasks provide alternate methods for running the tests.
 
+To execute both server and browser tests, but skip linting, use `npm run test:quick`.
+
+```bash
+npm run test:quick
+```
 
-`npm run test:quick`
-Runs both server and browser tests, but skips linting
+Use `npm run test:server` when you want to run only the server tests.
 
-`npm run test:server`
-Run only the server tests
+```bash
+npm run test:server
+```
 
-`npm run test:browser`
-Run only the browser tests. Coverage reports are available for browser tests by running `npm run test:coverage`. You can find the results under the `coverage/` directory that will be created upon completion.
+When you'd like to execute individual server-side test files, you can use the command below. This is great for development and for quickly identifying bugs.
```

Comment:
You might add that because this is using the local mocha, babel compilation will happen automatically without having to futz with mocha command line arguments. That's the main reason these npm tasks exist.